static/js/controllers/contacts-content.js

```diff
@@ -119,26 +120,57 @@ var _ = require('underscore');
         var split = _.partition(children, function(child) {
```

Comment:
Again not your code, but this seems quite an obscure way to sort an array!  Would have thought [`Array.prototype.sort()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) would do the job.